docs/user-documentation/glossary.md

```diff
@@ -1,95 +1,202 @@
- 
+Open source describes a method of software development that promotes access to the end product's source code. Islandora is an open source product with an active development community, operating under the [GPL license (2.0)](https://www.gnu.org/licenses/old-licenses/gpl-2.0.html) for [Drupal](#drupal) components and the [MIT license](https://opensource.org/licenses/MIT) for non-[Drupal](#drupal) components. 
+
+## PR
+See [Pull request](#pull-request)
+
+## Pull request
+Also PR; sometimes also known as merge requests; technical term from distributed version control systems for software code like Git. Code contributors can request that the maintainer of a code repository 'pulls' the code change into the repository after approval.
+
+## RDF
+See [Resource Description Framework](#resource-description-framework)
+
+## Repository Item
+See: [Resource Node](#resource-node)
+
+## Resource Description Framework
+Also RDF; family of World Wide Web Consortium (W3C) specifications originally designed as a data model for metadata. It has come to be used as a general method for conceptual description or modeling of information that is implemented in web resources. The data is modelled as a set of statements, also known as triples. A collection of RDF statements intrinsically represents a directed graph. Data represented according to the RDF specifications can be serialized in different ways, for instance using [JSON-LD](#json-ld).
+
 ## Resource Node
-A resource node is a Drupal node that represents a single conceptual item or object stored in an Islandora repository. It acts as a container for all files and metadata associated with that item, and is the place where the item 'lives' as a visitable url. 
+A resource node is a [Drupal](#drupal) [Node](#node) that represents a single conceptual item or object stored in an Islandora repository. It acts as a container for all files and metadata associated with that item, and is the place where the item 'lives' as a visitable URI. 
  
-For example, A video stored in Islandora will have a resource node, which metadata stored in fields, a preservation master file, and may have a thumbnail, web-friendly derivatives, and technical metadata associated with the resource node. The resource node may also belong to one or more collections. 
+For example, a video stored in Islandora will have a resource node, with metadata stored in [Fields](#field). Attached to the resource node is a [Media](#media) entity, which encapsulates the preservation-grade file. The Resource node may be linked to further [Media](#media), for instance for a thumbnail, web-friendly derivative, and technical metadata associated with the resource node. The resource node may also belong to one or more collections. 
+
+## Taxonomy
```

Comment:
The core Drupal module that adds functionality for managing [Taxonomy terms](#taxonomy-term) in [Vocabularies](#vocabulary)